tests/integration-tests/tests/common/mpi_common.py

```diff
@@ -60,7 +60,7 @@ def _test_mpi(
 
     if verify_scaling:
         assert_scaling_worked(
-            scheduler_commands, region, stack_name, scaledown_idletime, expected_max=3, expected_final=0
+            scheduler_commands, region, stack_name, scaledown_idletime, expected_max=5, expected_final=2
```

Comment:
why is the Max and final node counts changing?

Reply:
Because in the cluster config file there are two queues that have a MinCount of 1, so the expect_final count of compute nodes will never go below 2. So the increase in number of computes nodes stays the same (3) because 3 dynamic nodes were added, that is added on top of the 2 static nodes that already exist.